release-notes/v1.2.0.yaml

```diff
@@ -0,0 +1,149 @@
+  Added support for a native prometheus metrics endpoint in the ratelimit server
+
+# Fixes for bugs identified in previous versions.
+bug fixes: |
+  Fixed xDS translation failed when wasm http code source configured without sha
+  Fixed unsupported listener protocol type causing an error while updating Gateway Status
+  Fixed some status updates were being discarded by the status updater
+  Fixed Gateway crash adding BackendTLSPolicy to External Backend of an HTTPRoute
+  Fixed Delay in SecurityPolicy change propagation for HTTPRoute when using targetSelectors
+  Fixed JSONPath not correctly translated to JSONPatch paths
+  Fixed allow empty slowStart when using LeastRequest
+  Fixed Backends which should be rejected are still used as an HTTPRoute's destination
+  Fixed losing timeout settings that originate from the route when translating the backend traffic policy
+  Fixed Backend resources don't get status updates
+  Fixed Active Health check requires expectedStatuses field to work
+  Fixed HTTPHeaderFilter processing doesn't correctly support multiple header values
+  Fixed multiple reference grants in same namespace
+  Fixed upstream get unwanted /.
+  Fixed creation of SecurityPolicy with targetSelectors fails
+  Fixed wrong gateway is chosen as HTTPRoute parent
+  Fixed override issue for EEP
+  Fixed nil pointer err translating hash load balancing
+  Fixed ratelimit does not work across multiple GatewayClasses
+  Fixed upstream mTLS only works for HTTPS listeners
+  Fixed nil pointer if backedtls.minVersion is set but backedtls.maxVersion is not
+  Fixed empty connection limit causes xDS rejection
+  Fixed ratelimit not working with both headers and cidr matches
+  Fixed EDS didn't update when deployments was created after services
+  Fixed RBAC issue for deleting infrastructure resources
+  Fixed customized infrastructure resources not being deleted
+  Fixed Gateways never become ready/programmed when running Envoy as a Daemonset
+  Fixed Ratelimit Deployment ignoring pod labels and annotation merge
+  Fixed the API Server receives unnecessary requests
+  Fixed terminating envoy pods don't respond with "Connection: close" (H1) or GOAWAY(H2) on shutdown, switch to an immediate drain strategy
+  Fixed ratelimit statsd not working
+  Fixed not generating selector of deployment/daemonset based on the custom label configuration of EnvoyProxy
+  Fixed egctl experimental translate using a wrong ns
+  Fixed reconcile not triggered for Secret updates referenced by a BackendTLSPolicy
+  Fixed Route with multiple parents has incorrect namespace in parentRef status
+  Fixed only log endpoint configuration in verbose logging mode (`-v 4` or higher)
+  Fixed the xDS translation failed when wasm http code source configured without a sha
+  Fixed HTTPRoute status only shows one parent when targeting multiple Gateways from different GatewayClasses
+  Fixed Route with multiple parents has incorrect namespace in parentRef status
```

Comment:
Fixed Route with multiple parents having an incorrect namespace in the parentRef status.